src/controllers/hid/hidcontroller.cpp

```diff
@@ -256,8 +256,8 @@ bool HidController::poll() {
     while (true) {
         // Cycle between buffers so the memcmp below does not require deep copying to another buffer.
         unsigned char* pPreviousBuffer = m_pPollData[m_iPollingBufferIndex];
-        m_iPollingBufferIndex = (m_iPollingBufferIndex + 1) % kNumBuffers;
-        unsigned char* pCurrentBuffer = m_pPollData[m_iPollingBufferIndex];
+        const int nextBufIndex = (m_iPollingBufferIndex + 1) % kNumBuffers;
```

Comment:
```suggestion
        const int nextBufferIndex = (m_iPollingBufferIndex + 1) % kNumBuffers;
```
Please spell out words